tools/aa/commands/WorkspaceCommand.java

```diff
@@ -121,6 +135,16 @@ public boolean run(String[] args) {
         System.err.println(RED_ERROR + "Workspace does not exist");
         return false;
       }
+
+      if (remove.get()) {
+        Path currentPath = Paths.get("").toAbsolutePath();
+        if (currentPath.startsWith(workspacePath)) {
+          System.err.println(RED_ERROR + "Trying to remove active workspace");
```

Comment:
Why not allow removing active workspace?
In bash, you can delete the current folder.

Reply:
@oferb but is this a feature and not a bug? 🙂 
apart from `cd`-ing away, you won't be able to do much being left in this state